### ISSUE TITLE
Embeddings for a single image

### ISSUE THREAD

**Issue by rishiraicom** [davidsandberg/facenet#321]
I am new to deep learning. I wanted to calculate embeddings for a single person in the image like this.
```
 feed_dict = { images_placeholder: image , phase_train_placeholder:False}
                emb = sess.run(embeddings, feed_dict=feed_dict)
```
here image only contain a single image.

but this results in the same embeddings for different people when running with single image.
The model i am using is the pre-trained model 

> 20170216-091149

**Comment by vudung45** [comment]
You probably didnt import the model correctly

**Comment by rishiraicom** [comment]
Thanks @vudung45, I have imported the model properly but the problem is that the embedding values keep improving as the number of photos of different people increase but when I give a single image as input it gives wrong embeddings

**Comment by vudung45** [comment]
@rishiraicom do you have skype or anything? email me at: davidvu98@gmail.com. I can try to help u out.

**Comment by zo7** [comment]
It looks like you're using an older model – try downloading one of the new ones on the README and see if the problem persists. There might be an issue with batch norm, which may be why you're having problems getting a good prediction for a single image. (See #234)

**Comment by gopinath-balu** [comment]
@vudung45 I have the same problem while working with your implementation. I'm just trying the run the model with the new pretrained weight provided here. Could you please shed some light ??
